posts/app.py

```diff
@@ -227,7 +226,7 @@ def __getattribute__(self, _):
     return pymongo.MongoClient(mongodb_uri).posts_db.posts_collection
 
 
-POSTS_COLLECTION = connect_to_mongodb()  # None if can't connect
+app.config["COLLECTION"] = connect_to_mongodb()
```

Comment:
Just wondering, was changing this for testing convenience or did the original way cause problems?

Reply:
A bit of both. This makes it easier to test, and setting global configurations like the active database in Flask is preferred to use `app.config` (just a dict with a few special keys like `"TESTING"`) instead of global variables which are less clean.